setup.cfg

```diff
@@ -44,6 +44,9 @@ zip_safe = False
 [options.packages.find]
 where = src
 
+[options.extras_require]
+docs = sphinx; sphinxcontrib-spelling; furo
```

Comment:
I don't see the need for a "docs" extra, who would use it? Nothing in this PR uses it.